migration/migration-technical-details.md

```diff
@@ -45,33 +45,40 @@ The registration window takes place within a specific range of blocks. The count
-6. The activation tool monitors readiness transactions within a 24 hour window.
-7. If 80% of the total stake (represented by registered validators) signals that they are ready by sending readiness transactions within 24 hours, the PoS chain will use the produced genesis block (with a block height of 3 456 000). The process automatically moves to the next activation window if the threshold is not reached.
-8. If the 80% readiness threshold is not met within the first 24 hours, a new activation window begins, starting with a new candidate block at the block immediately following 1440 blocks or 24 hours from the first candidate block. Validators need to send the readiness transactions again for the new genesis hash produced with this new transition candidate block.
-9. In subsequent activation windows, the same exact process is repeated. Select a new candidate block 24 hours apart from the previous one by opening a new activation window until the PoS chain starts
-10. Once 80% readiness is reached, the PoS client starts, and the once candidate block becomes the transition block, marking the start of the Nimiq PoS blockchain.
+The activation phase follows a specific sequence of events, outlined in the following list. The overall process will be explained in more detail afterward.
+1. **3 days** before November 19th, validators running the activation tool will begin sending automatic **online transactions** every hour before the first candidate block is mined to signal their online status.
+2. The candidate block is mined at the block height 3,456,000, officially opening the first activation window.
+3. The network waits for 10 confirmations after the candidate block to prevent block reversion. 
+4. After the 10 confirmations, pre-registered validators capture the state at block 3,456,000 and begin generating the PoS "genesis" block. Validators generate the "genesis" block deterministically. The block hash generated should be the same across all validators.
+5. After generating the "genesis" block, validators send their readiness transactions, which include the block hash in the data field.
+6. The activation tool monitors readiness transactions within a 24-hour window.
+7. If 80% of the total stake (represented by registered validators) sends readiness transactions within 24 hours, the PoS chain starts at block height 3,456,000, disregarding any blocks mined after this point.
+8. If the 80% readiness threshold is not met within the first 24 hours, a new activation window begins, starting with a new candidate block 1,440 PoW blocks (~24 hours) after the first.
+9. Validators who do not send readiness transactions by the **5th activation window** will be removed and deactivated from the first epoch of the PoS chain. This process ensures that inactive validators are removed from consideration to the required threshold.
```

Comment:
Starting the 5th activation window, those validators who are not ready, will no longer be considered part of the readiness voting process and will be deactivated from the first epoch of the PoS chain.This process ensures that inactive validators are removed from consideration to the required threshold.